Source/Public/Restore-Bicep.ps1

```diff
@@ -0,0 +1,85 @@
+<#
+.SYNOPSIS
+    Restore bicep module dependencies, like dotnet restore for bicep.
+.DESCRIPTION
+    When using bicep modules in a project, it is likely that the same module
+    is reused in several projects. For example a module for creating a KeyVault.
+
+    When improving a module that is used in several places, updating it everywhere
+    quickly becomes a hassle. This is where Restore-Bicep is helpful!
+
+    Instead of manually copying the correct version of a module to every place it
+    is used, simply create a JSON file along with your bicep-file that declares any
+    module dependencies, thier version and which repository they are available at.
+    
+    Before running bicep build, run Bicep-Resore and point it to the JSON file
```

Comment:
Typo on row 15, should be Bicep-Restore. ` Before running bicep build, run Bicep-Resore and point it to the JSON file`